### ISSUE TITLE
CMake install remaps target directory name to install space. URDF includes cannot find files.

### ISSUE THREAD

**Issue by hsu** [NASA-JSC-Robotics/val_description#15]
For example, [model/materials/ is installed to ${CATKIN_PACKAGE_SHARE_DESTINATION}/materials](https://github.com/NASA-JSC-Robotics/val_description/blob/develop/CMakeLists.txt#L77). Doing so might make it hard to switch development when working within the devel and install spaces.

For example, looking at [valkyrie_A_sim.xacro](https://github.com/NASA-JSC-Robotics/val_description/blob/develop/model/robots/valkyrie_A_sim.xacro), the includes look like below:
~~~
    <xacro:property name="mesh_root" value="package://val_description/model/meshes"/>
~~~
If one were operating out of the install target, then this will have to be changed to
~~~
    <xacro:property name="mesh_root" value="package://val_description/meshes"/>
~~~
Which can be very confusing.

**Comment by jlack1987** [comment]
Ah ok I see what you mean. I will take care of this.

**Comment by jlack1987** [comment]
Sorry about this, my bad. Should be fixed now!

**Comment by scpeters** [comment]
Fixed by 5b99c9134bc5159ce647462dbe825efc7f393fd9 I believe.
